influxql/_v1tests/server_test.go

```diff
@@ -0,0 +1,6396 @@
+package v1tests
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/influxdata/influxdb/v2/models"
+	"github.com/influxdata/influxdb/v2/tests"
+)
+
+const (
```

Comment:
In the OSS context, can / should we:
* Stop skipping any of these categories of tests?
* Delete the tests labeled with any of these categories?

Reply:
I don’t want to diverge them in this PR, as it has been copied over verbatim. It makes sense to do a careful review of the skipped tests (some may be from OSS 1.x) and prepare a separate set of issues / PRs for cloud and OSS to address them.

Reply:
Makes sense to me!